source/dayTime.cpp

```diff
@@ -0,0 +1,66 @@
+#include "../headers/dayTime.h"
+#include <iostream>
+
+int DayTime::stringToDays(string input) {
```

Comment:
I don't understand why we need the stringToX functions if we have a standardized input scheme there should be no way for random strings of unknown formats to be floating around in the backend. All the inputs should be sanitized by the time they get back there.

Reply:
So mw, tr, 8am, 9.40am, 11.20am, 1pm, 2.40pm and 4.20pm are already converted to their respective array locations before arriving here?

Reply:
They are all input as a 0 or 1, available or unavailable. See the Example CSV input files in our drive. There is a specific input for each time. Thus we don't need the labels. (In the case of instructor it is 0 for unavailable, 1 for can teach but prefer not to, and 2 for prefer to teach at this time.)